docs/cardinal/game/configuration.mdx

```diff
@@ -0,0 +1,41 @@
+| CARDINAL_NAMESPACE           | The name of your cardinal game shard. If you are running multiple game shards in a single World Engine instance, this MUST be unique for each game shard. Must only be alphanumeric characters.                      | YES                     |
+| CARDINAL_MODE                | The cardinal mode determines how your game shard runs. Must be either "development" or "production". In development, Cardinal will not submit any transactions to the base shard, and security concerns are lowered. | N/A                     |
+| REDIS_ADDRESS                | The address of the redis instance.                                                                                                                                                                                   | YES                     |
+| REDIS_PASSWORD               | The password of your redis instance.                                                                                                                                                                                 | YES                     |
+| CARDINAL_LOG_LEVEL           | The level of logging Cardinal should output. See [Leveled Logging](https://github.com/rs/zerolog?tab=readme-ov-file#leveled-logging) for more details.                                                               | NO                      |
+| BASE_SHARD_SEQUENCER_ADDRESS | The address of your EVM Base Shard deployment. This will typically be running on port 9601.                                                                                                                          | YES                     |
+
+## EVM Base Shard `[evm]`
+| Name            | Description                                                                                                                                                                                           |
+|-----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| DA_AUTH_TOKEN   | The authentication token required to communicate with the Celestia DA Layer. This is typically generated by the celestia light client, and can be found in the logs during startup                    |
+| DA_BASE_URL     | The base URL of the DA layer.                                                                                                                                                                         |
+| DA_NAMESPACE_ID | The namespace the EVM will post its blobs under. Learn more about DA namespaces here [Namespaces](https://celestia.org/glossary/namespaced-merkle-tree/) Can be generated with `openssl rand -hex 10` |                                                                                                                 |
```

Comment:
There's missing punctuation before "Can be generated...".

Reply:
fixed [nits](https://github.com/Argus-Labs/world-engine/pull/736/commits/4e49fffad477551015c40df5da722fea99730694)